### ISSUE TITLE
Expanding / restoring model/house preview resets the sequencer layout. 

### ISSUE THREAD

**Issue by sidelights** [smeighan/xLights#3279]
**Is your feature request related to a problem? Please describe.**
Noticed in .12 that when maximizing / restoring the house preview windows after adjusting the top divider resets the divider back to the layout prior to the maximize. 

**Describe the solution you'd like**
It seems logical that if I have just adjusted the sizing of sub windows max/restoring a preview should remember what my prior change. 

**Describe alternatives you've considered**
saved my current view after making the layout adjustment making sure it's my current view, and then max/restore didn't seem to make a difference. 

**Additional context**
opening window view:

![startWindow](https://user-images.githubusercontent.com/101691917/172784654-993b6091-ab0c-4e03-978e-a96197d0eb8f.png)

drag top frame down to see effects settings for example:
![pull window down](https://user-images.githubusercontent.com/101691917/172785378-cfdaa5aa-c0d3-42c2-baca-a0bfdf2e3c6f.png)

max model preview window, then restore to prior value using double box icon next to X. 

![max model preview_then min window again](https://user-images.githubusercontent.com/101691917/172786140-63bc0331-bc98-4c48-8f36-7697724b8199.png)

Divider has returned to the original location, prior to the adjustment preceding the max / min. 

![startWindow](https://user-images.githubusercontent.com/101691917/172786569-8c737734-2ba2-467c-8dc7-667b15eaccee.png)

**Comment by computergeek1507** [comment]
Did you try creating a perspective to save/load your windows layout?

**Comment by sidelights** [comment]
Yes, saving a new perspective, updating an existing perspective, or default perspective does not seem to make a difference.  Also does not appear to matter if you have a sequence open or not.  Just need to be in the sequencer tab. 
Issue occurs on windows / mac, confirmed by at least 4 members of the zoom room today.  at least 2 mac and 2 windows tests.

https://user-images.githubusercontent.com/101691917/172916940-cf467ca9-48b0-4491-93c2-0e8c41baad47.mp4

**Comment by derwin12** [comment]
I dont see us "fixing" this .. when you maximize a window - it will resize the layout of the remaining windows into defaults. This is a wxWidgets function. One would have to store every window height/width and restore them on a max/min operation. It would likely break more things than it fixes.
